dockstore-client/src/test/java/io/dockstore/client/cli/LaunchTestIT.java

```diff
@@ -1352,4 +1352,26 @@ public void malJsonToolWdlLocal() {
         );
         runClientCommand(args, false);
     }
+    @Test
+    public void provisionInputWithPathSpaces() {
+        //Tests if workflow runs when json points to an input file path that contains spaces
+        File helloWDL = new File(ResourceHelpers.resourceFilePath("helloSpaces.wdl"));
+        File helloJSON = new File(ResourceHelpers.resourceFilePath("helloSpaces.json"));
+
+        ArrayList<String> args = new ArrayList<String>() {{
+            add("--local-entry");
+            add("--json");
+            add(helloJSON.getPath());
+        }};
+
+        WorkflowsApi api = mock(WorkflowsApi.class);
+        UsersApi usersApi = mock(UsersApi.class);
+        Client client = new Client();
+        client.setConfigFile(ResourceHelpers.resourceFilePath("config"));
```

Comment:
Which file is this?

Reply:
the json points to a hello.txt file with spaces in directory path, the wdl basically just cats the contents of the hello.txt file

Reply:
I mean the config file

Reply:
https://github.com/ga4gh/dockstore/blob/develop/dockstore-client/src/test/resources/config

Reply:
Now that I think about it, is this necessary with a local entry? I based this test on the other tests with mocks and they many seem to grab the config despite being local entries.

Reply:
ya...a lot of our tests are not quite correct.  That config file isn't even a CLI config file.  Which means running this test will set server-url to Dockstore production and probably hit some endpoints.  Pick another config file to use.